src/cve/pipeline/pipeline.py

```diff
@@ -119,14 +119,14 @@ def convert_input_to_df(message: AgentMorpheusEngineInput) -> ControlMessage:
             len(v.vuln_package_intel_sources) > 0 for v in message.info.vulnerable_dependencies
         ]
 
-        has_sufficient_intel_flags = full_df["has_sufficient_intel"]
+        has_sufficient_intel_flags = full_df["has_sufficient_intel_for_agent"]
 
         full_df["vulnerable_dependencies"] = vulnerable_dependencies
 
         # Filter full_df by whether the CVE has vulnerable dependencies or lacks vulnerable package info from intel
```

Comment:
This comment needs an update after this change
```suggestion
        # Filter full_df for conditions that should skip the agent
```